test/src/api/commands/web-element/testIsVisisble.js

```diff
@@ -0,0 +1,138 @@
+const assert = require('assert');
+const {WebElement} = require('selenium-webdriver');
+const MockServer = require('../../../../lib/mockserver.js');
+const CommandGlobals = require('../../../../lib/globals/commands-w3c.js');
+const common = require('../../../../common.js');
+const Element = common.require('element/index.js');
+
+describe('element().isVisible() command', function() {
+  before(function(done) {
+    CommandGlobals.beforeEach.call(this, done);
+  });
+
+  after(function(done) {
+    CommandGlobals.afterEach.call(this, done);
+  });
+
+  it('test .element().isVisible()', async function() {
+    MockServer.addMock({
+      url: '/session/13521-10219-202/element/0/property/value',
+      method: 'GET',
+      response: JSON.stringify({
+        value: ''
+      })
+    },
+    true
+    );
+
+    const resultPromise = this.client.api.element('#signupSection').getValue();
```

Comment:
Just copy-pasting the tests won't do, please make changes at the required places accordingly.